nilearn/plotting/surface/_utils.py

```diff
@@ -87,16 +91,16 @@ def _check_bg_map(bg_map, hemi):
 
 
 def _get_hemi(mesh, hemi):
-    """Check that a given hemisphere exists in a PolyMesh and return the
-    corresponding mesh. If "both" is requested, combine the left and right
-    hemispheres.
+    """Check that a given hemisphere exists in a
+    :obj:`~nilearn.surface.PolyMesh` and return the corresponding mesh. If
+    "both" is requested, combine the left and right hemispheres.
     """
     if hemi == "both":
         return combine_hemispheres_meshes(mesh)
     elif hemi in mesh.parts:
         return mesh.parts[hemi]
```

Comment:
```suggestion
    elif hemi in ["left", "right"]:
        if hemi in mesh.parts:
            return mesh.parts[hemi]
        else:
            raise ValueError(f"hemi = {=hemi} does not exist in mesh. Avaliable hemispheres are: {mesh.parts.keys()}")
```

Reply:
Does this seem reasonable?

Reply:
yup but I think you will want:

` ValueError(f"{hemi=} does not exist in mesh. Avalaible hemispheres are: mesh.parts.keys()}")`